addons/payment_ogone/controllers/main.py

```diff
@@ -0,0 +1,129 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import base64
+import logging
+import pprint
+import re
+
+import werkzeug
+
+from odoo import _, http
+from odoo.exceptions import ValidationError
+from odoo.http import request
+from odoo.tools import ustr
+
+_logger = logging.getLogger(__name__)
+
+
+class OgoneController(http.Controller):
+    _flexcheckout_return_url = '/payment/ogone/flexchekout'
```

Comment:
Isn't there a typo on the last flexcheckout word?

Reply:
There is, indeed! It shall be fixed soon.